docs/index.rst

```diff
@@ -0,0 +1,24 @@
+.. include:: ../README.rst
+
+Welcome to Lumache's documentation!
```

Comment:
This feels like a really odd transition without context on the index page. Should this be on another subpage that we link to from the `Rendered` docs?

Reply:
I kind of agree... it's an attempt to have an example project while at the same time showing how we would document a fictive software project. So this content would normally be in the index.rst page?